src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/OverrideMarkerEntity.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.personrecord.jpa.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import jakarta.persistence.Version
+import uk.gov.justice.digital.hmpps.personrecord.model.types.OverrideMarkerType
+
+@Entity
+@Table(name = "override_marker")
+class OverrideMarkerEntity(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
+
+  @ManyToOne(optional = false)
```

Comment:
I have no context here at all so feel free to disregard this.
At first glance I would have imagined this to be a column on either Person orPersonKey

Reply:
My impression is that we are only unmerging record (i think) which has a knock on effect of creating new uuids potentially. One to double check. Hence why i've only done it at record level for now

Reply:
OK, as long as you are sure that it is a ManyToOne relationship, given that it is at record (person) level. Surely a person is either included or excluded (or neither).

I am aware I am coming in like a seagull on this, sorry about that. A lot going on on PiC at the moment, just trying to help.